storage/disk/example_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package disk_test
```

Comment:
> it can be used as a library along with the rego package.

Demonstrating that here would be nice, I think.

Reply:
Added an example test to the rego package.